mvnw

```diff
@@ -0,0 +1,2 @@
+set -x
+mvn -f proxy $@
```

Comment:
general question, what is this file for? I don't see it used in other file here.

Reply:
is for travisCI, but will remove it, doesn't make sense